### DISCUSSION TITLE
no such file or directory

### DISCUSSION THREAD

**Discussion by thezem** [vercel/vercel#8389] [Help] [answered]
can anyone please tell a working way to just read a file in nodejs serverless functions

here's what've tried already 

`path.join(process.cwd(),'body', 'index.css')`
`path.resolve(process.cwd(),'body', 'index.css')`
`path.resolve(__dirname,'body', 'index.css')`
`path.join(process.cwd(), 'body', 'index.css')`
path.join(__dirname, 'body', 'index.css')`

And.. 
`readFileSync('./body/index.css')
`

![image](https://user-images.githubusercontent.com/65074998/184425751-d8a2c3f9-6410-44e4-9cfa-23e09307837b.png)

**Comment by amyegan** [comment, marked as answer]
_See answer in reply https://github.com/vercel/vercel/discussions/8389#discussioncomment-3387156_

Thanks for reaching out @antiihope! It looks like your code is trying to access a file that doesn't exist. Maybe the file is being deployed to a different location than you expected? I'm guessing that you're trying to do something similar to the examples for [how to use files in serverless functions](https://vercel.com/support/articles/how-can-i-use-files-in-serverless-functions). 

 If you can share your repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I would be happy to take a closer look.

**Reply by thezem** [reply]
Hi! thank you for helping me out , found out i was adding 'utf8' to the path resolve and not the readFile function , the problem is fixed now: thank you!
